test/streams.js

```diff
@@ -1,4 +1,4 @@
-const Trie = require('../src/index.js')
+const Trie = require('../dist/index.js').CheckpointTrie
 const describe = require('tape')
 
 describe('kv stream test', function (tester) {
```

Comment:
Can we have the `require` statements consistent without the `.js` appendix - like e.g. `const ScratchDB = require('../dist/scratch').ScratchDB`? Otherwise test file changes look all ok.